main.go

```diff
@@ -52,14 +47,14 @@ func main() {
 	controllerInitializers := app.DefaultInitFuncConstructors
 
 	fss := cliflag.NamedFlagSets{}
-	command := app.NewCloudControllerManagerCommand(ccmOptions, cloudInitializer, controllerInitializers, fss, wait.NeverStop)
+	command := app.NewCloudControllerManagerCommand(ccmOptions, cloudInitializer,
```

Comment:
I would like to automated these formatting tasks. I tried `golines`, but it fails for some lines. See: https://github.com/segmentio/golines/issues/93

Do you know a way to format the above code lines automatically?

Reply:
If I use `golines` only on main.go, it would create this output:

```
diff --git main.go main.go
index f4af909..3bb6e25 100644
--- main.go
+++ main.go
@@ -47,13 +47,20 @@ func main() {
        controllerInitializers := app.DefaultInitFuncConstructors
 
        fss := cliflag.NamedFlagSets{}
-       command := app.NewCloudControllerManagerCommand(ccmOptions, cloudInitializer,
-               controllerInitializers, fss, wait.NeverStop)
+       command := app.NewCloudControllerManagerCommand(
+               ccmOptions,
+               cloudInitializer,
+               controllerInitializers,
+               fss,
+               wait.NeverStop,
+       )
        code := cli.Run(command)
        os.Exit(code)
 }
 

```

I commit above code.

Is this format ok for you?

Reply:
yes